build.gradle

```diff
@@ -124,17 +124,19 @@ def generate_distro_tasks = { dist, gPackageName, gInputFile, gVersion, gRelease
             include 'conf/**'
             eachFile { details ->
                 def pkgPath = details.path - 'conf'
-                details.path = pkgPath 
+                details.path = pkgPath
+                configurationFile(details.path)
             }
         }
         from(tar){
             fileType CONFIG | NOREPLACE
```

Comment:
From what I gather, the fileType parameter works for RPM packaging, so I left it in in-case it's needed here.